quickstart/RELEASING.md

```diff
@@ -0,0 +1,22 @@
+
+# Releasing a new version of the Quickstart
+
+The Quickstart lives in the `/quickstart` directory of the Ziti repo. It is a set of scripts and docker images that are used in the doc site to help users get started with Ziti. The Quickstart is also used by the Ziti team to test Ziti releases.
```

Comment:
You don't need to explain the quickstart imo, just how it gets released and when (which is done below) and how. Since this md is so short, you could even jsut slap it into the readme if you want. ?